app/views/hours/show.html.slim

```diff
@@ -3,13 +3,14 @@
     .table-responsive
       table.table.table-no-border-top
         tbody
-          tr
-            - if current_user.superadmin?
+          - if current_user.superadmin?
+            tr
               td= t_model(Volunteer)
               td= @hour.volunteer.contact.full_name
-          tr
-            td= t_model(Client)
-            td= @hour.assignment.client.contact.full_name
+          - if @hour.assignment?
+            tr
+              td= t_model(Client)
+              td= @hour.hourable.client.contact.full_name
```

Comment:
Else shouldn't it show the corresponding group offer?

Reply:
this is only to show informations about the client which is not included in a group offer

Reply:
Sure, I meant and in case the `@hour` is referring to a group offer shouldn't it show info for the group offer? So
`- if @hour.assignment?
  .... (as it is now)
- else
  `title of the group offer`

Reply:
Sure, I meant and in case the @hour is referring to a group offer shouldn't it show info for the group offer? So
`- if @hour.assignment?`
`.... (as it is now)`
`- else`
`title of the group offer`

Reply:
added :)